README.md

```diff
@@ -30,14 +30,14 @@ Amazon Simple Notification Service (SNS) to send SMS.
    - oneVizionUrl - OneVizion URL
    - oneVizionLogin - OneVizion Login
    - oneVizionPwd - OneVizion Password
-   - serviceId - ID of the Notification Service createf at step 2
+   - serviceId - ID of the Notification Service created at step 2
    - phoneNumberField - The name of the field which contains the phone number of the recepient. Recepient is a user trackor related with User ID from the Notif Queue record.
```

Comment:
'Rec**E**pient' should be replaced with 'Rec**I**pient';
Line 29 has a typo 'integ**AR**tion'

Reply:
Fixed